backend/src/commands/populate_database.ts

```diff
@@ -0,0 +1,65 @@
+import sequelize from 'src/models';
+sequelize;
+import User from 'src/models/User';
+import Category from 'src/models/Category';
+import Video from 'src/models/Video';
+import Author from 'src/models/Author';
+
+const NAMES = ['tom', 'benny', 'roland'];
+
+function populateUsers () {
+  console.log('POPULATING USERS');
```

Comment:
I think we should start with an ESLint config soon to prevent console log statements in production code. :D